CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Changelog
 
+## 0.4.27
+
+This release includes support for the `[dependency-groups]` table as recently standardized in [PEP 735](https://peps.python.org/pep-0735/). The table allows for declaration of optional dependency groups that are not published as part of the package metadata, unlike `[project.optional-dependencies]`. There are new `--group`, `--only-group`, and `--no-group` options throughout the uv interface.
```

Comment:
Instead of "options throughout the uv interface."... "flags throughout" to avoid overloading "options"?

Reply:
I use "flags" for boolean CLI components that do not take values and "options" for CLI components that take values. I try to be consistent about this in the documentation. I'm hesitant to switch here, but don't feel strongly if you found it confusing as-is.